oojs.md

```diff
@@ -24,13 +34,19 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+class Album extends Playlist={
+
+  artist:function(artist){
+    this.artist=artist;
+  }
+  Album.push(artist)
```

Comment:
What is this line supposed to be doing?

Remember, you cannot have free-floating code outside of the class methods.